config/application.rb

```diff
@@ -29,7 +29,6 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
```

Comment:
Starting a new review by commenting on this line again...I screwed up the last one. Can we remove this comment?